manifest_staging/charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -73,7 +73,8 @@ spec:
         - --mutating-webhook-configuration-name={{ .Values.mutatingWebhookName }}
         - --external-data-provider-response-cache-ttl={{ .Values.externaldataProviderResponseCacheTTL }}
         - --experimental-enable-k8s-native-validation={{ .Values.enableK8sNativeValidation }}
-        - --vap-enforcement={{ .Values.vapEnforcement }}
+        - --default-create-vap-for-templates={{ .Values.createVAPForTemplates }}
+        - --create-vap-binding-for-constraints={{ .Values.createVAPBindingForConstraints }}
```

Comment:
please add default- in front of both of these flags for vap and vapb. and for helm values, pls also prepend with default.